Libnd4j: "mvn clean" fails on Windows cmd

Not a big deal (we can - and I usually do) run clean in msys shell. But it would be nice to fix this at some point.

```
[INFO] ------------------------------------------------------------------------
[INFO] Building libnd4j 1.0.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ libnd4j ---
[INFO]
[INFO] --- javacpp:1.4.1:build (javacpp.cppbuild.clean) @ libnd4j ---
[INFO] Detected platform "windows-x86_64"
[INFO] Building for platform "windows-x86_64"
[INFO] rm -Rf blasbuild
[ERROR] Failed to execute JavaCPP Builder: Cannot run program "rm" (in directory "C:\DL4J\Git\deeplearning4j\libnd4j"): CreateProcess error=2, The system cannot find the file specified
```

--- a/libnd4j/pom.xml
+++ b/libnd4j/pom.xml
@@ -128,20 +128,25 @@
                             <workingDirectory>${project.basedir}</workingDirectory>
                         </configuration>
                     </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>2.6</version>
+                <executions>
                     <execution>
                         <id>javacpp.cppbuild.clean</id>
                         <phase>clean</phase>
                         <goals>
-                            <goal>build</goal>
+                            <goal>clean</goal>
                         </goals>
                         <configuration>
-                            <skip>${clean.skip}</skip>
-                            <buildCommand>
-                                <program>rm</program>
-                                <argument>-Rf</argument>
-                                <argument>blasbuild</argument>
-                            </buildCommand>
-                            <workingDirectory>${project.basedir}</workingDirectory>
+                            <filesets>
+                                <fileset>
+                                    <directory>blasbuild</directory>
+                                </fileset>
+                            </filesets>
                         </configuration>
                     </execution>
                 </executions>

